config/locales/notices/en.yml

```diff
@@ -34,9 +34,8 @@ en:
     use_diff_email:
       link: use a different email address
       text_html: Or, %{link}
-  session_expired_html: Please %{link} the page to log in.
-  session_expired_link: refresh
   session_timedout: >
     We signed you out due to inactivity. This helps keep your account safe.
     Please sign in again.
   session_timeout_warning: Your session will end in %{time_left_in_session} due to inactivity.
+  session_cleared: We cleared your information for your security.
```

Comment:
Also, telling the user that we cleared their information when they might not have typed anything at all is confusing to me.